optimizely/event/event_processor.py

```diff
@@ -159,9 +159,9 @@ def _get_time(self, _time=None):
       Integer time in seconds.
     """
         if _time is None:
-            return int(round(time.time()))
+            return time.time()
```

Comment:
Please change the description of the method.
Also Returns, valid time, can be float and integer. Need to revise documentation.